components/region_cache_memory_engine/src/background.rs

```diff
@@ -481,20 +598,32 @@ pub struct BackgroundRunner {
 
     gc_range_remote: Remote<yatp::task::future::TaskCell>,
     gc_range_worker: Worker,
+
+    // Region load and eviction worker.
+    // TODO: this can be consolidated, possibly with the GC worker.
+    load_evict_remote: Remote<yatp::task::future::TaskCell>,
+    load_evict_worker: Worker,
 }
 
 impl Drop for BackgroundRunner {
     fn drop(&mut self) {
         self.range_load_worker.stop();
         self.delete_range_worker.stop();
         self.gc_range_worker.stop();
+        self.load_evict_worker.stop();
     }
 }
 
+// The expected average region size in bytes: used to estimate the number of top
+// regions to cache (using `soft limit threshold /
+// `EXPECTED_AVERAGE_REGION_SIZE`)`.
+pub const EXPECTED_AVERAGE_REGION_SIZE: usize = 96_000_000;
```

Comment:
The region size is configured by parameters, it can be not 96MB.

Reply:
Good call. Looking at it piping this in through config now.

Reply:
Done. PTAL at how I changed this.